FinniversKit/Sources/Components/Label/Label.swift

```diff
@@ -74,3 +74,28 @@ extension Label {
         UIMenuController.shared.setMenuVisible(true, animated: true)
     }
 }
+
+// MARK: - Support HTML
+
+public extension Label {
+    func setText(fromHTMLString htmlString: String, style: [String: String] = [:]) {
+        var styledText = htmlString
+        for (itemIndex, styleItem) in style {
+            styledText = styledText.replacingOccurrences(of: itemIndex, with: styleItem)
+        }
+        let htmlTemplate = "<span style=\"font-family: \(font!.fontName); font-size: \(font!.pointSize); color: \(textColor.hexString)\">\(styledText)</span>"
```

Comment:
We shouldn't need to force unwrap `font` for `Label`/`UILabel`, so I think this should be removed